src/components/LoanCollections/HomeExp/KivaLoanCardCategory.vue

```diff
@@ -0,0 +1,137 @@
+<template>
```

Comment:
This component is supposed to have a carousel for the loan cards on small screens. This currently hides all the loans by default, only showing loans for medium-and-up screen sizes.